Misc/NEWS.d/next/Core_and_Builtins/2025-05-18-14-33-23.gh-issue-69605.ZMO49F.rst

```diff
@@ -0,0 +1 @@
+Stop import names completion fallback on regular names completion in the :term:`REPL`.
```

Comment:
Perhaps we can make this a bit clearer? Can't think of a better wording right now though..